racetrack_client/README.md

```diff
@@ -1,30 +1,93 @@
-# Racetrack client context
-`racetrack-client` is a CLI client tool for deploying workloads to Racetrack.
+# Racetrack CLI client
+`racetrack-client` is a CLI client tool for deploying and managing workloads in Racetrack.
 
 Racetrack system allows to deploy jobs in a one step.
 It transforms your code to in-operation workloads, e.g. Kubernetes workloads.
 You write some code according to a set of coventions, you include the manifest file which explains the code, 
 and you submit it to Racetrack. A short while after, the service calling your code is in operation.
 
-# Installation
+## Quickstart
+```shell
+# Install racetrack-client
+pip3 install racetrack-client
+
+# Set current remote
```

Comment:
Do we explain anywhere what a remote is? We can't assume they have that idea already from git.

Reply:
Good idea. I've explained what alias and remote are:

> Alias is a short, friendly name for the URL of your Racetrack server, which is also known as "remote".
From now on, you can refer to your remote with an alias.